### PR TITLE
add Collection ordinal-moopy

### PR DESCRIPTION
**Pull request by HARU-moopy**

### CODE REVIEW

**Comment by hosein6666** [comment]
Ok

**Comment by hosein6666** [comment]
Yes

File: collections/Ordinal-moopy/meta.json
Comment (hosein6666) [inline collections/Ordinal-moopy/meta.json:1]: Hoseinghasemi

Comment (hosein6666) [reply collections/Ordinal-moopy/meta.json:1]: Hoseinghasemi

**Comment by HARU-moopy** [comment]
Corrected uppercase letters to lowercase.
Thank you for your confirmation.

**Comment by HARU-moopy** [comment]
Ordinal-moopy
↓
ordinal-moopy

Fix completed.

**Comment by HARU-moopy** [comment]
Ordinal-moopy
↓
ordinal-moopy

Fix completed.

**Pull request closed without merging**